src/Text/KirbyTags.php

```diff
@@ -33,7 +34,12 @@ public static function parse(string $text = null, array $data = [], array $optio
         return preg_replace_callback($regex, function ($match) use ($data, $options) {
             try {
                 return static::$tagClass::parse($match[0], $data, $options)->render();
+            } catch (InvalidArgumentException $e) {
```

Comment:
Why is the InvalidArgumentException in particular still swallowed? I think it could be useful to throw all Exceptions in debug mode to make them easy to debug. But maybe I just don’t see the use-case for this exception (pun intended). 😅

Reply:
I was trying hard, to avoid this pun.. glad you did it tho 😂
About the Exception and its use case is: It is not unlikely to have potential KirbyTags in your content, which are not defined and therefore should be just ignored. This currently works, by throwing an InvalidArgumentException: https://github.com/getkirby/kirby/blob/21ebebda6bdde9aee62140f81f658f0852ea4e60/src/Text/KirbyTag.php#L42
It might be pretty distracting to have those errors be thrown, even in debug mode, because it is expected behavior. On the other hand, this Exception is widely used throughout the app and it is indeed not ideal to swallow them all. Might be a good idea to a new specific Exception Class here instead, what do you think?

Reply:
That makes sense! 👍

I don’t think we should add another Exception class as we would end up with potentially hundreds of classes for different cases.

What we could so is to check for the Exception message and only swallow that exact Exception.

As I’m going to work on the tests anyway, I will implement that today or tomorrow.